aws-integration/trace-sample-app/values.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    io.kompose.service: sample-app
+  name: sample-app
+spec:
+  ports:
+    - name: "4567"
+      port: 4567
+      targetPort: 4567
+  selector:
+    io.kompose.service: sample-app
+status:
+  loadBalancer: {}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    io.kompose.service: sample-app
+  name: sample-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: sample-app
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        io.kompose.service: sample-app
+    spec:
+      containers:
+        - env:
+            - name: AWS_REGION
+              value: ${AWS_REGION}
+            - name: LISTEN_ADDRESS
+              value: 0.0.0.0:4567
+            - name: OTEL_EXPORTER_OTLP_ENDPOINT
+              value: http://otel-collector.${KYMA_NS}:4317
```

Comment:
This basically bypasses Kyma's telemetry component. The idea behind our otel collector deployments is that developers have a well defines, Kyma specific OTLP endpoint where to send the data to. On the other end, the backend (in this case the self-deployed collector) should be configured only on cluster level and be hidden from the applications. You should use Kyma's services to ingest metrics and traces. The Otel SDK should have two settings for metrics and traces.